build-images.sh

```diff
@@ -69,6 +70,9 @@ if ! buildah containers --format "{{.ContainerName}}" | grep -q nodebuilder-samb
     buildah from --name nodebuilder-samba -v "${PWD}:/usr/src:Z" docker.io/library/node:lts
 fi
 
+echo "Downloading user manager ${user_manager_version} UI..."
+curl -O -L https://github.com/NethServer/ns8-user-manager/releases/download/${user_manager_version}/ns8-user-manager-${user_manager_version}.tar.gz
```

Comment:
If there's no `-f` arg, curl will download a `Not found` response, that will be loaded in to the image as is

Are you sure you want to keep it that way?